bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+# Set up Rails app. Run this script immediately after cloning the codebase.
+
+# Exit if any subcommand fails
+set -e
+
+# Set up ruby version file
+if [ ! -f .ruby-version ]; then
+  cp .ruby-version.example .ruby-version
+fi
+
+# Set up configurable environment variables
+if [ ! -f .env ]; then
+  cp config/env .env
+fi
+
+# Set up Ruby dependencies via Bundler
+gem install bundler --conservative
+bundle check || bundle install --path=vendor/bundle
```

Comment:
> when run without bundle exec, which can cause unexpected and broken behavior.

Only if you're not expecting it, and if you don't have the CLI gems installed globally it's not an issue.

> other reasons

It makes it easy to delete a project directory when you're done (or want a fresh start) without leaving old/unused gems lying around on the system.